shared/index.d.ts

```diff
@@ -989,12 +1071,74 @@ declare interface SettingsSlider extends Panel {
 	RestoreCVarDefault(): void;
 }
 
+declare interface SettingsKeyBinder extends Panel {
+	bind: string;
+
+	OnShow(): void;
+}
+
+declare interface SettingsToggle extends Panel {
+	convar: string;
+
+	/** Set the cvar back to default */
+	RestoreCVarDefault(): void;
+
+	OnShow(): void;
+}
+
+declare interface SettingsEnum extends Panel {
+	convar: string;
+}
+
+declare interface SettingsEnumDropDown extends Panel {
+	convar: string;
+
+	OnShow(): void;
+
+	RefreshDisplay(): void;
+
+	/** Set the cvar back to default */
+	RestoreCVarDefault(): void;
+}
+
 declare interface AvatarImage extends Panel {
 	accountid: string;
 	
 	steamid: string;
 }
 
+declare interface BaseBlurTarget extends Panel {
+	/**
+	 * Add a panel to the blur list
+	 * @param panel 
+	 */
+	AddBlurTarget(panel: Panel);
+
+	/**
+	 * Remove a panel from the blur list
+	 * @param panel 
+	 */
+	RemoveBlurPanel(panel: Panel);
+}
+
+declare interface TripleMonitorBackground extends Panel {
+}
+
+interface ClockTypeEnum {
+	NONE: 'none',
+	REALTIME: 'realtime',
+	WALL: 'wall',
+	GAME: 'game',
+	GAMETICK: 'game-tick',
+	GAMESERVER: 'game-server'
+}
```

Comment:
This needs a `@group` annotation so that it'll show up correctly in the docs.
If this is globally defined in code, I suggest redefining this with `declare enum`. (see `p2ce/weapons.d.ts`)
If it isn't, then a `ClockType` type should be defined so we can use it as a value type. (see `shared/index.d.ts`)
ex.
```ts
/** @group enum */
declare enum ClockTypeEnum {
// ...
}
```
or
```ts
declare type ClockType = ValueOf<ClockTypeEnum>;
/** @group enum */
inferface ClockTypeEnum {
// ...
}
```